docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/token/create-native-token-foundary.md

```diff
@@ -12,59 +12,82 @@ tags:
 
 import ExampleCodeIntro from '../../../_partials/how-tos/token/_example_code_intro.md';
 
-# Create a Native Token Foundary 
+# Create a Native Token
 
-You can use the new `createNativeTokenFoundry` method to streamline creating and managing native tokens. This single method consolidates the functionalities of minting native tokens and registering them as ERC20 tokens. Two separate methods previously handled this process: 
-`mintNativeTokens` and `registerERC20NativeToken`. 
-By using `createNativeTokenFoundry`, you can efficiently mint new tokens and register them for use as ERC20 tokens in one seamless operation, simplifying the overall token management workflow. 
-This method ensures that only the foundry owner can mint tokens, maintaining security and control over the token creation process.
+This guide will show you how you can efficiently mint new tokens and register them for use as ERC20 tokens with the `createNativeTokenFoundry` function in one seamless operation. It will create a foundry on L1 and registers it as a ERC20 on L2. This method ensures that only the foundry owner can mint tokens, maintaining security and control over the token creation process.
```

Comment:
```suggestion
This guide will show you how you can efficiently mint new tokens and register them for use as ERC20 tokens with the `createNativeTokenFoundry` function in one seamless operation. It will create a foundry on L1 and register it as an ERC20 on L2. This method ensures that only the foundry owner can mint tokens, maintaining security and control over the token creation process.
```